corehq/apps/sms/api.py

```diff
@@ -550,7 +562,7 @@ def process_sms_registration(msg):
 
 def incoming(phone_number, text, backend_api, timestamp=None,
              domain_scope=None, backend_message_id=None,
-             raw_text=None, backend_id=None):
+             raw_text=None, backend_id=None, media_url=None):
```

Comment:
Is there a possibility that there could be more than one media URL? Would it make sense to have this be a list with zero or more items?

Reply:
Currently multimedia is implemented for infobip backend only. Infobip webhook payload contains a list of messages and each message can have only one media_url link. Since we are already looping through all messages and calling the incoming method for each of them, I would say it is not necessary to convert it to a list. However some other backend may have different implementation so I am going to convert this into a list anyway.

Reply:
Ok so I've converted it to a list. Thanks for the suggestion.